### PR TITLE
fixing typo for one of the challenges in es6 section for issue #34578

### LINKED ISSUE #34578: Typing error on "ES6: Create an Export Fallback with export default" lesson.
**Describe your problem and how to reproduce it:**
The test case instruction has a very minor typo error. It say ""Proper used of export fallback".
However this should be "Proper use of export fallback".

**Add a Link to the page with the problem:**
https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/es6/create-an-export-fallback-with-export-default/

**Tell us about your browser and operating system:**
* Browser Name: Google Chrome
* Browser Version: 70.3
* Operating System: MacOS

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
screenshot 2018-12-07 at 4 41 01 am

### PR DESCRIPTION
**Pull request by sirdarthvader**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [ x] My article does not contain shortened URLs or affiliate links.

Closes #34578
screenshot 2018-12-07 at 4 41 01 am

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-an-export-fallback-with-export-default.english.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The following function should be the fallback value for the module. Please add t
 
 ```yml
 tests:
-  - text: Proper used of <code>export</code> fallback.
+  - text: Proper use of <code>export</code> fallback.
     testString: getUserInput => assert(getUserInput('index').match(/export\s+default\s+function\s+subtract\(x,y\)\s+{return\s+x\s-\s+y;}/g), 'Proper used of <code>export</code> fallback.');
```
Comment (Nirajn2311) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-an-export-fallback-with-export-default.english.md]: Could you also change the typo in the `testString`

```suggestion
     testString: getUserInput => assert(getUserInput('index').match(/export\s+default\s+function\s+subtract\(x,y\)\s+{return\s+x\s-\s+y;}/g), 'Proper use of <code>export</code> fallback.');
```

**Review by Nirajn2311 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-an-export-fallback-with-export-default.english.md (hunk 1 above)
Comment (ashishcodes4) [reply curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-an-export-fallback-with-export-default.english.md]: Have added a new commit, should be completed, sorry about the miss.

**Review by Nirajn2311 — APPROVED** [review]

**Comment by ashishcodes4** [comment]
When will this be pushed to the live application, I am new to this, this is my first pull request for a live application so I am not sure how this really works around.

**Review by ezioda004 — APPROVED** [review]

**Review by ezioda004 — COMMENTED** [review]
On second thought, I think we can improve this tests verbiage.
Perhaps something like:
>Your code used <code>export</code> fallback.

@ashishcodes4 First your PR needs to be merged and then we'll be pushing master to production, thats when the changes will be live.

**Comment by procode200** [comment]
This problem got me confused as well.
Can I help fix it?

Hang 10 ;)

**Comment by ashishcodes4** [comment]
I think the issue is sorted now.

**Comment by ezioda004** [comment]
@ashishcodes4 Refer to my review above ☝️

**Comment by procode200** [comment]
@ezioda004 No Thanks!

**Comment by ashishcodes4** [comment]
So are we making any further  changes to this or not!

**Comment by ashishcodes4** [comment]
So are we making any further changes to this or not ? @ezioda004 ???

**Comment by ezioda004** [comment]
@ashishcodes4 We are. (Check my previous comment)
@procode200 If you dont have anything meaningful to add to this discussion, I'd suggest not commenting.

**Comment by ashishcodes4** [comment]
@ezioda004 I am not sure if that exact verbiage "Your code used export fallback."  is the correct way of saying it, I think the way it is now is better.
What do you think?

**Comment by ezioda004** [comment]
@ashishcodes4 Going by the other challenges we have - for example [this](https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/functional-programming/add-elements-to-the-end-of-an-array-using-concat-instead-of-push) one. The error message is more readable and expressive.

Thats why I think:
>Your code should use `export` fallback.

Would be more fitting.

**Comment by ashishcodes4** [comment]
Have changed the verbiage and added the commit. Let me know if this does the work.

**Review by ezioda004 — APPROVED** [review]

**Pull request merged**